episodes/03-audience.md

```diff
@@ -98,13 +203,14 @@ before they can follow your lesson.
 
 If you are struggling with this exercise because your lesson audience is novices,
 think about skills like touch typing, using a web browser, or interacting with a command line 
-or graphical interface.  These are skills experts or competent practitioners can commonly overlook.
+or graphical interface. 
+These are skills commonly overlooked by experts and competent practitioners.
```

Comment:
We could have an additional exercise here "testing prerequisite knowledge" where they would try to come up with a quiz.

Reply:
There is already an exercise along these lines, for the participants working alone on their lessons at least. See [lines 81-84 of `03-audience.md`](https://github.com/carpentries/lesson-development-training/blob/8dad5b6de5f7a983859eaef6ac9f920d4aae4dcf/episodes/03-audience.md?plain=1#L81)

Reply:
Yes, I noticed that - was wondering if all learners should have a go at this. But this may make an already long lesson too long.